tests/Garm64-test-sve-signal.c

```diff
@@ -96,8 +101,22 @@ int64_t square(svint64_t z0)
   return res;
 }
 
+bool has_sve(void) {
+#if defined(__linux__)
+  return (getauxval(AT_HWCAP) & HWCAP_SVE) ? true : false;
+#else
+  printf("Cannot determine if SVE is present, assuming it is\n");
+  return true;
```

Comment:
It would be better to assume  the OS does *not* support SVE. That way the test will SKIP instead of crash and FAIL and if a port wants it to PASS they need to fix the test. SKIP means "not supported" and "FAIL" means "supported but broken".

Reply:
Done